### PR TITLE
feat(charts): added `DonutChart`

### LINKED ISSUE #991: Add `DonutChart`
### Description

`DonutChart` is based on [PieChart recharts component](https://recharts.org/en-US/api/PieChart).

### Problem Statement/Justification

Depending on the project, it may be necessary to have a function that makes it easy to understand numbers on dashboards, etc.

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by 108yen**
Closes #991

## Description

Added new chart component.

## Current behavior (updates)

NIL

## New behavior

`DonutChart` is available

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by koralle** [comment]
@108yen 
I thought it would be good to have a prop that controls the anchor point for determining the position to display the Tooltip (top-left, top-right, bottom-right, or bottom-left).

For instance, if top-right is specified, the top-right corner of the Tooltip will align with the mouse hover position.

スクリーンショット 2024-04-01 21 43 31

**Review by suzukisan22 — COMMENTED** [review]
not sure about whole package, but here are something i can review...

**Comment by 108yen** [comment]
@koralle 
Unfortunately, that functionality is not provided by rechart.

File: packages/components/charts/src/chart-tooltip.tsx [hunk 1]
```diff
@@ -14,8 +14,15 @@ export const ChartTooltip = forwardRef<ChartTooltipProps, "div">(
...
-      ({ color, name, value: valueProp } = {}, index) => {
+      ({ color: colorProp, name, value: valueProp, ...rest } = {}, index) => {
         let value: string
+        let color: string
+
+        if (colorProp) {
+          color = colorProp
+        } else {
+          color = rest.payload.color
+        }
```
[the remark this answers is not in the document]
Comment (108yen) [reply packages/components/charts/src/chart-tooltip.tsx]: Fixed！

**Comment by suzukisan22** [comment]
when i access local storybook, accessibility violation founded. it should be fixed.
http://localhost:6006/?path=/story/components-feedback-donutchart--basic

**Comment by 108yen** [comment]
@suzukisan22 
Could you share those screenshots?

**Comment by 108yen** [comment]
It is withdrawn because the function was forgotten to be implemented.

**Pull request closed without merging**